test/Fail/Issue1977.err

```diff
@@ -1,5 +1,5 @@
-Issue1977.agda:8.7-22: error: [NoParseForApplication]
-Could not parse the application [ M.undefined ]
-Operators used in the grammar:
-  M.[_] (closed operator) [[_] (Issue1977.agda:6.5-8)]
-when scope checking [ M.undefined ]
+Issue1977.agda:8.7-8: error: [NotInScope]
+Not in scope:
+  [
+  at Issue1977.agda:8.7-8
+when scope checking [
```

Comment:
This feels like a real behavioral change.
The old parser seemed to have taken `[_]` into account even though it was only in the scope as qualified, but then decided that it cannot use it after all.
The new parser does not consider `[_]` in the first place.
This seems to be a better behavior, and it fixes #1977 properly.